packages/api/src/user.js

```diff
@@ -218,6 +218,11 @@ export async function userTokensGet (request, env) {
           name
           secret
           created
+          uploads(_size: 1) {
+            data {
+              _id
+            }
+          }
```

Comment:
Instead of returning this array of a single upload with an ID, why don't we transform it into a boolean property that's more useful to the UI?:

```js
res.findAuthTokensByUser.data = res.findAuthTokensByUser.data.map(t => {
  t.hasUploads = Boolean(t.uploads.data.length)
  delete t.uploads
  return t
})
```

Reply:
Good suggestion, added :)